documentation/config_file_puppetserver.markdown

```diff
@@ -94,6 +96,8 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 jruby-puppet: {
     ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby]
     gem-home: /opt/puppetlabs/server/data/puppetserver/jruby-gems
+    environment-vars: { "HOME" : ${HOME}
+                        "LANG" : "de_DE.UTF-8" }
```

Comment:
@camlow325 used ${HOME} to show how HOCON can use expansion and "LANG" to show that it can be hardcoded, although I'm not sure if it's desirable for users to do that